src/ui/components/LedgerViewer.js

```diff
@@ -51,6 +57,13 @@ const useStyles = makeStyles((theme) => {
   };
 });
 
+const DATE_SORT = {
+  name: "Date",
+  fn: (e: LedgerEvent) => e.ledgerTimestamp,
+};
+
+const PAGINATION_OPTIONS = [25, 50, 100];
```

Comment:
nit: i suggest `deepFreeze`ing so that the defaults can't get modified.

Reply:
accepted